capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './baseDecorator'
```

Comment:
- [ ] Well on the effort you put into this project. However, after it, I have found that there are a few things that we should address to get everything as expected.

1. I think the path to the decorator class is not correctly set up. Please let's fix that. 

-   Instead of this :point_right:  `require_relative './baseDecorator'` do this :point_right: `require_relative './base_decorator'` because the `name` of the file where we have the `BaseDecorator` class is `base_decorator` instead of `baseDecorator`.